CampaignConfig/mdc2020_prolog.cfg

```diff
@@ -45,7 +45,7 @@ generate-email-summary = True
 expected-lifetime = 23h
 memory = 2500MB
 maxConcurrent = 10000
-email-to = srsoleti@fnal.gov
+email-to = sophie@fnal.gov
```

Comment:
Rob, can we define an email alias prodmanager@fnal.gov that can be re-assigned to whoever is the current manager?

Reply:
Yes, this is something I've been meaning to point out. If Rob could do that, it would be great!

Reply:
This has been resolved